src/SFA.DAS.EAS.Web/DependencyResolution/DefaultRegistry.cs

```diff
@@ -14,16 +15,16 @@ public DefaultRegistry()
         {
             Scan(s =>
             {
-                s.AssembliesFromApplicationBaseDirectory(a => a.GetName().Name.StartsWith("SFA.DAS"));
+                s.AssembliesFromApplicationBaseDirectory(a => a.GetName().Name.StartsWith("SFA.DAS") && !a.GetName().Name.StartsWith("SFA.DAS.Authorization"));

```

Comment:
does this need to be in here anymore?

Reply:
Not sure about this @tom-gough , but it's the same in `SFA.DAS.EmployerAccounts.Web` as well. We can, maybe, remove it from both. Not sure what the purpose of it it is/was.

Reply:
Yes otherwise it auto registers all of the classes in the auth package which goes horribly, horribly wrong.